CMakeLists.txt

```diff
@@ -47,7 +47,15 @@ else()
 endif()
 
 ## Add serial library
+set(WJWSERIAL_BUILD_SHARED "ON" CACHE BOOL "Only build the shared version.")
```

Comment:
why not just ```option(BUILD_SHARED_LIBS "Only build the shared version." False)```?

Reply:
Because I wanted it to build both by default. As this PR obviously not going I will close it.